.changeset/fast-chefs-invent.md

```diff
@@ -0,0 +1,5 @@
+---
+"@latticexyz/world": major
```

Comment:
I think this can be `minor` since its not breaking?
```suggestion
"@latticexyz/world": minor
```

Reply:
`contractToInterface` signature, and `worldgen` output changed, which is kind of breaking, but I have no strong opinion here